### ISSUE TITLE
False positive constraint-missing-not-valid when adding primary key constraint using index.

### ISSUE THREAD

**Issue by dreid** [sbdchd/squawk#693]
Adding a primary key constraint using an existing index gets flagged by `constraint-missing-not-valid`.

See this example taken from the documentation for [`adding-serial-primary-key-field`](https://squawkhq.com/docs/adding-serial-primary-key-field)

```SQL
ALTER TABLE account ADD CONSTRAINT account_pk PRIMARY KEY USING INDEX account_pk_idx;
```

[Playground link](https://play.squawkhq.com/#code/IIGQKgogSgBGwCEQRgQwMboPYFcB2ALjMACIkwDCA8gHIDKYUwAkjWGproQPoAOA1jAAKUZgFlgUAJowA0hBkBVOqwDiMViQgANDtnwE+-bgEsAJgA8A3ACggA)

I notice there is code in the rule for handling unique constraints, https://github.com/sbdchd/squawk/blob/master/crates/squawk_linter/src/rules/constraint_missing_not_valid.rs#L122

I assume this just needs to also handle primary key constraints.

**Comment by sbdchd** [comment]
Fixed in the latest version `v2.29.0`!

Thank you for the detailed report!
